plugins/source/aws/client/services.go

```diff
@@ -878,7 +878,14 @@ type ServiceCatalogAppRegistryClient interface {
 //go:generate mockgen -package=mocks -destination=./mocks/ses.go . SESClient
 type SESClient interface {
 	GetEmailTemplate(ctx context.Context, params *sesv2.GetEmailTemplateInput, optFns ...func(*sesv2.Options)) (*sesv2.GetEmailTemplateOutput, error)
-	ListEmailTemplates(ctx context.Context, params *sesv2.ListEmailTemplatesInput, optFns ...func(*sesv2.Options)) (*sesv2.ListEmailTemplatesOutput, error)
+	sesv2.ListEmailTemplatesAPIClient
+	GetConfigurationSet(ctx context.Context, params *sesv2.GetConfigurationSetInput, optFns ...func(*sesv2.Options)) (*sesv2.GetConfigurationSetOutput, error)
+	sesv2.ListConfigurationSetsAPIClient
+	GetConfigurationSetEventDestinations(ctx context.Context, params *sesv2.GetConfigurationSetEventDestinationsInput, optFns ...func(*sesv2.Options)) (*sesv2.GetConfigurationSetEventDestinationsOutput, error)
+	sesv2.ListContactListsAPIClient
+	GetContactList(ctx context.Context, params *sesv2.GetContactListInput, optFns ...func(*sesv2.Options)) (*sesv2.GetContactListOutput, error)
+	sesv2.ListEmailIdentitiesAPIClient
+	GetEmailIdentity(ctx context.Context, params *sesv2.GetEmailIdentityInput, optFns ...func(*sesv2.Options)) (*sesv2.GetEmailIdentityOutput, error)
```

Comment:
Super annoying that they only expose APIClients for the List APIs, but can we organize them alphabetically?

Reply:
Sorted in [`7f7e279` (#3475)](https://github.com/cloudquery/cloudquery/pull/3475/commits/7f7e27986713691062d70aee4b8087a12c5ddcde) and I can't agree more. The APIClient is exposed only if it has the paginator I think.